configure.py

```diff
@@ -1665,7 +1665,7 @@ def MatchingFor(*versions):
     ActorRel(Matching,    "d_a_obj_homensmoke", extra_cflags=['-pragma "nosyminline on"']),
     ActorRel(NonMatching, "d_a_obj_hsehi1"),
     ActorRel(NonMatching, "d_a_obj_htetu1"),
-    ActorRel(NonMatching, "d_a_obj_iceisland"),
+    ActorRel(Matching, "d_a_obj_iceisland", extra_cflags=['-pragma "nosyminline on"']),
```

Comment:
There's some kind of version difference, if you don't know what it is you can mark it matching for only JPN and PAL so the USA build doesn't fail:
```suggestion
    ActorRel(MatchingFor("GZLJ01", "GZLP01"), "d_a_obj_iceisland", extra_cflags=['-pragma "nosyminline on"']),
```